### ISSUE TITLE
UAC management for an app build with electron-packager

### ISSUE THREAD

**Issue by codisart** [codisart/Edith-Path#11]
Hi

I used the winreg npm package for my app and I need elevated privileges for the uses I have.

I would like to associate a manifest file with an app build with electron (atom-shell) and packaged on windows with electron-packager but I can not find any way to do it easily.

An old app of mine was made with cx_freeze and and it sufficed to put the file with the name app.exe.manifest in the same directory.

I can't understand why it doesn't work this time.

**Comment by codisart** [comment]
Check node-runas

**Comment by codisart** [comment]
Hi,

I solved this problem.
So first to know is an external manifest won't count if an embedded manifest is already present. (see answer to http://superuser.com/questions/905103/what-does-it-mean-to-enable-windows-to-prioritise-external-manifests)
And it turns out that the electron.exe has already a manifest embedded and the package done with electron-package is mostly a rename of the electron executable.
So in order to embed a news manifest with the right privileges needed, I took the manifest in the electron repo (https://github.com/atom/electron/blob/master/atom/browser/resources/win/atom.manifest), edited the security section and used the mt.exe tool to add the new manifest https://msdn.microsoft.com/en-us/library/aa375649%28v=vs.85%29.aspx

And tadaa !
